src/planwise/client/db.cljs

```diff
@@ -28,7 +28,10 @@
 
    ;; Projects
    :projects
-   {:creating? false}
+   {:creating? false
+    :creating-waiting? false
+    :loading? false
+    :current nil}
```

Comment:
Cache of `id => project` was replaced by a single `current` field holding the project being shown